pkg/generator/helper/helper.go

```diff
@@ -31,10 +31,13 @@ type stepInfo struct {
 	StepFunc         string
 	StepName         string
 	StepSecrets      []string
+	Containers       []config.Container
+	Sidecars         []config.Container
+	Outputs          config.StepOutputs
 }
 
 //StepGoTemplate ...
-const stepGoTemplate = `// Code generated by piper's step-generator. DO NOT EDIT.
+var stepGoTemplate = `// Code generated by piper's step-generator. DO NOT EDIT.
```

Comment:
Could you elaborate why this is necessary?

Reply:
probably leftover from temporary refactoring, thanks for your attention.